io.github.Archeb.opentrace.yml

```diff
@@ -0,0 +1,70 @@
+app-id: io.github.Archeb.opentrace
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+separate-locales: false
+rename-icon: opentrace
+rename-desktop-file: opentrace.desktop
+command: OpenTrace
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Flatpak
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: opentrace
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+    build-commands:
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 opentrace.png -t /app/share/icons/hicolor/scalable/apps
+      - install -Dm755 opentrace.sh /app/bin/OpenTrace
+      - install -Dm644 opentrace.desktop -t /app/share/applications
+      - install -Dm755 apply_extra -t /app/bin
+      - install -Dm755 nexttrace.sh -t /app/bin/
+      - install -Dm755 /usr/bin/desktop-file-edit -t /app/bin/
```

Comment:
why do you need that?

Reply:
desktop-file-edit is an dependency from where I copied this config file :( Now thats removed